andela/AndelaEatsUI
# #160536333 Implement admin delete meal

**What does the PR do?**
- Implements `Delete Meal Item` functionality

**Description of Task to be completed?**
- Ensures an admin can delete a meal item

**How should this be manually tested?**
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
-  cd into AndelaEatsUI and checkout to this branch `ft-implement-admin-delete-meal-160536333`
- run `npm run bundle` and `npm run dev`
- Log in as an `Admin`, navigate to `admin dashboard`, select `Meals`, and click on `delete`
- run `npm test` to ensure all tests are passing

**What are the relevant pivotal tracker stories?**
- [#160536333](https://www.pivotaltracker.com/story/show/160536333)

**Screenshots**
<img width="1677" alt="screen shot 2018-09-21 at 9 22 06 pm" src="https://user-images.githubusercontent.com/26305570/45904430-e8291600-bde4-11e8-8a4b-f3f432ff83b7.png">
<img width="1680" alt="screen shot 2018-09-21 at 9 22 38 pm" src="https://user-images.githubusercontent.com/26305570/45904457-055de480-bde5-11e8-8d10-414bc6cdfe50.png">
